src/modules/logger/logged_topics.cpp

```diff
@@ -62,9 +62,9 @@ void LoggedTopics::add_default_topics()
 	add_topic("cpuload");
 	add_optional_topic("esc_status", 250);
 	add_topic("failure_detector_status", 100);
-	add_topic("follow_target_status", 100);
 	add_topic("follow_target", 500);
 	add_topic("follow_target_estimator", 100);
+	add_topic("follow_target_status", 100);
```

Comment:
Before merging we should probably decrease the logging rate to 200ms or even 400ms. For debugging it was nice to have the high rate.